### DISCUSSION TITLE
Is there any way to add additional attributes to SwiftUI elements?

### DISCUSSION THREAD

**Discussion by jamesvanhorn** [appium/appium#18601] [Q&A]
I realize this is less of an Appium question and really a Swift / XCUI test question, but trying here as well as there. 

Would be great to be able to expose some extra info about an element to automated testing while not exposing it to the end user via VoiceOver. Wondering if anyone here found a solution for that.

**Comment by robhav** [comment]
You can use accessibilityIdentifiers. They will not be read by VoiceOver.
https://developer.apple.com/documentation/uikit/uiaccessibilityidentification/1623132-accessibilityidentifier
